### PR TITLE
Initial Solana Migration and Usage Documentation

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

File: docs/solana/rewardable_entities.mdx [hunk 1]
```diff
@@ -70,27 +73,28 @@ While this may look difficult, this is all neatly packed in the Metaplex Digital
...
+entities for a given wallet is to filter by the `helium-entity-manager-creator`
 
 We can get this address by running
 
 ```js
-import { entityCreatorKey } from "@helium/helium-entity-manager-sdk";
+import { entityCreatorKey } from '@helium/helium-entity-manager-sdk'
 
-const hntMint = new PublicKey("...hnt mint...")
+const hntMint = new PublicKey('...hnt mint...')
```
Comment (jthiller) [inline docs/solana/rewardable_entities.mdx]: I'm not sure why Prettier felt comfortable giving code suggestions...

File: docs/solana/rewardable_entities.mdx [hunk 2]
```diff
@@ -403,6 +409,5 @@ const tx = await formTransaction({
   lazyDistributor: lazyDistributorPkey,
 })
 
-await provider.send(tx);
+await provider.send(tx)
```
Comment (jthiller) [inline docs/solana/rewardable_entities.mdx]: Prettier again 🤔

Comment (samgutentag) [reply docs/solana/rewardable_entities.mdx]: think ive got a fix for these, adding `<!-- prettier-ignore -->` before a code block makes prettier ignore the code block

example:

carbon

File: docs/solana/rewardable-entities.mdx [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+
+The documentation on Solana and the upcoming Migration is under development and subject to change.
+
+:::
+
+# Rewardable Entities (Hotspots, Mappers, and more)
+
+Following the transition to Solana, all actors in the network that receive tokens are considered "Rewardable Entities."
+
+While, as of writing this document, hotspots are the only rewardable entity, this will change with the addition of mappers and other rewardable categories.
```
Comment (mbthiery) [inline docs/solana/rewardable-entities.mdx]: Will staking and HSTs not be implemented at launch as rewardable entities? If not how will they be implemented?

Comment (ChewingGlass) [reply docs/solana/rewardable-entities.mdx]: Fanout wallet, using Hydra. See metaplex program library

File: docs/oracles/data-transfer-oracles.mdx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+:::warning 🚧 This document is a work in progress. 🚧
+
+The documentation on Solana and the upcoming Migration is under development and subject to change.
+
+:::
+
+Moving the entire data transfer logic off-chain will increase scalability and reduce points of
+failure.
+
+## Lorawan Network Server Streamlining
```
Comment (abhay) [inline docs/oracles/data-transfer-oracles.mdx]:
```suggestion
## LoRaWAN Network Server Streamlining
```

File: docs/oracles/data-transfer-oracles.mdx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+On the Helium L1, being a LoRaWAN Network Server (LNS) on the Helium Network requires running the
+router, [state channels](/blockchain/packet-purchasing/#state-channels), and the Helium chain, a
+significant hurdle for non-blockchain native LNS configurations.
+
+On the Solana L1, the LNS will have a much-reduced role. After purchasing their Organizationally
```
Comment (abhay) [inline docs/oracles/data-transfer-oracles.mdx]:
```suggestion
On the Solana L1, the LNS will have a focused role. After purchasing their Organizationally
```

File: docs/oracles/iot-proof-of-coverage-oracles.mdx [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+description: Helium IOT Proof of Coverage Oracles Documentation
+image: https://i.imgur.com/bWAbESE.png
+slug: /oracles/iot-proof-of-coverage-oracles
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+:::warning 🚧 This document is a work in progress. 🚧
+
+The documentation on Solana and the upcoming Migration is under development and subject to change.
```
Comment (abhay) [inline docs/oracles/iot-proof-of-coverage-oracles.mdx]: some of this stuff is active now. might be worth highlighting the parts that are.

File: docs/solana/migration/exchange.mdx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+list of Solana transactions and then compressed into a merkle root on Solana. This means that anyone
+can execute the transactions to inflate a wallet so long as they provide a proper proof that the
+provided transactions are valid. Each transaction is only allowed to be executed once.
+
+We can see one such transaction here:
+
+https://explorer.solana.com/tx/5Y8vfMAfhyqgXsp1YnvPVHMdK4PUryLqMCgGynQXg9tPkUVehK438XviXz3ciSkg4PhqMZ8UyrSSVj7pgY8uq1xr?cluster=devnet
+
+If you would like a copy of the database serving the proofs for the migration service, please
+contact the helium foundation. You can then run your own instance of the
```
Comment (abhay) [inline docs/solana/migration/exchange.mdx]:
```suggestion
[contact the Helium Foundation](hello@helium.foundation). You can then run your own instance of the
```

Comment (abhay) [reply docs/solana/migration/exchange.mdx]: We should probably also just publish the proofs in shadow or arweave and link here.

**Review by abhay — DISMISSED** [review]

File: docs/oracles/iot-proof-of-coverage-oracles.mdx (hunk 6 above)
Comment (samgutentag) [reply docs/oracles/iot-proof-of-coverage-oracles.mdx]: on it! breaking into smaller PRs will make this easier

**Review by samgutentag — APPROVED** [review]
Work in Progress, let get it out there!

**Pull request merged**